pkgs/os-specific/linux/adaptivemm/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+let
+  pname = "adaptivemm";
+  version = "unstable-20230116";
+in stdenv.mkDerivation {
```

```suggestion
let
  pname = "adaptivemm";
  version = "unstable-2023-01-16";
in stdenv.mkDerivation {
```
And no reason to use let here.